polling_stations/settings/base.py

```diff
@@ -301,3 +302,5 @@
 # importing test settings file if necessary (TODO chould be done better)
 if len(sys.argv) > 1 and sys.argv[1] in ["test", "harvest"]:
     from .testing import *  # noqa
+
+CANONICAL_URL= ''
```

Comment:
I think this should line should be:
```python
CANONICAL_URL= 'https://wheredoivote.co.uk'
```
However this is all so that we can see how well our links are shared on fb, which I'm not sure we'll ever do. Anyway for now at least having the right url seems sensible.